tailwind.config.ts

```diff
@@ -120,5 +121,15 @@ export default {
 			},
 		},
 	},
-	plugins: [animatePlugin, radixPlugin],
+	plugins: [
+		animatePlugin,
+		radixPlugin,
+		plugin(function ({ addComponents }) {
+			addComponents({
+				'.floating-toolbar': {
+					'@apply flex items-center absolute left-3 right-3 bottom-3 p-4 pl-5 md:pl-7 rounded-lg shadow-accent shadow-xl gap-2 md:gap-4 bg-muted/80 backdrop-blur-sm': {},
+				},
+			})
+		}),
```

Comment:
I would much rather put this in a new file called `app/styles/global.css` which is imported as a [css side-effect import](https://remix.run/docs/en/1.18.1/guides/styling#css-side-effect-imports) and uses [CSS bundling](https://remix.run/docs/en/1.18.1/guides/styling#css-bundling)

Reply:
Gotcha! That works. I try to stay in "tailwind world" as much as I can, but I can how a side-loaded CSS file that might be simpler to manage and better-scoped, since you don't need it everywhere.

Reply:
I've made some changes of my own. I'm going to push and merge :D

Reply:
Awesome, sounds good!